openstack_exporter/collectors/cinderbackend.py

```diff
@@ -180,15 +205,19 @@ def calculate_capacity_factors(self,
     def _parse_pool_data(self, pool, volume_type):
         """Construct the data from the pool information from the scheduler."""
         caps = pool['capabilities']
+        shard_name = None
+        if 'vcenter-shard' in caps:
+            shard_name = caps['vcenter-shard']
```

Comment:
Could be `shard_name = caps.get('vcenter-shard')` if you want to.

Reply:
done